README.md

```diff
@@ -53,24 +95,44 @@ const response = await fetch(poetNodeUrl + '/works/', {
 })
 ```
 
-Note, if you are creating an identity claim, your Identity Provider (IDP) <!-- TODO: link to glossary --> will be the issuer of the claim. If you are self-serving your own IDP, you will have to create an IdentityClaim for the IDP from which you can issue all further identities.
+### Example 2: createClaim for Work Claim, with overriding context
+
+
+
+Note, if you are creating an identity claim, your IDP will be the issuer of the claim. Frost is one such IDP.
```

Comment:
Linking `Frost` to `https://frost.po.et/` I think might be nice?

Reply:
Good catch.